plugins/pa11y/readme.md

```diff
@@ -0,0 +1,45 @@
+# Description 
```

Comment:
**Request:** We should give this document a title that reflects the contents. Here is a suggestion inspired by the recent [Secret Squirrel plugin](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/secret-squirrel#dotcom-tool-kitsecret-squirrel):

```suggestion
# @dotcom-tool-kit/pa11y
```